pig-latin/tests/pigLatin.test.js

```diff
@@ -0,0 +1,41 @@
+import { translateToPigLatin } from "../pigLatin";
+describe("translateToPigLatin", () => {
+  it("translates words starting with different vowels correctly", () => {
+    expect(translateToPigLatin("apple")).toBe("appleway");
+    expect(translateToPigLatin("elephant")).toBe("elephantway");
+    expect(translateToPigLatin("orange")).toBe("orangeway");
+    expect(translateToPigLatin("iguana")).toBe("iguanaway");
+  });
+});
+describe("translateToPigLatin", () => {
+  it("translates words starting with one consonant correctly", () => {
+    expect(translateToPigLatin("banana")).toBe("ananabay");
+    expect(translateToPigLatin("tiger")).toBe("igertay");
+    expect(translateToPigLatin("zebra")).toBe("ebrazay");
+    expect(translateToPigLatin("lion")).toBe("ionlay");
+  });
+});
+describe("translateToPigLatin", () => {
```

Comment:
You can just use one describe block for this entire file. The tests are all testing the same function.